### ISSUE TITLE
Is it possible to merge "Allow to use proxy per subsystem" from original qbittorent?

### ISSUE THREAD

**Issue by swhoro** [c0re100/qBittorrent-Enhanced-Edition#480]
### Suggestion

Add proxy for different use
original qb has merged the pr: https://github.com/qbittorrent/qBittorrent/pull/18528

### Use case

In some regions/countries, some rss servers may be blocked.
In that case, I want rss use proxy but peer/torrent not use it.

**Comment by abcfy2** [comment]
Upstream does not release this yet (Maybe 4.6.0). So just wait for official released.

**Comment by zhangBouble** [comment]
same

**Comment by c0re100** [comment]
Possibly, but this PR is 4.6.0 milestone, I think it's hard to follow up after merge...

**Comment by c0re100** [comment]
I've merged few PR (Proxy PR included) to [another branch](https://github.com/c0re100/qBittorrent-Enhanced-Edition/tree/v4_5_merge).
But I don't have proxy configure in qBittorrent, so I can't test it.
Let's try~

Windows x64: https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.5.3.10/test_v4_5_merge_Win_x64.7z

**Comment by AlphonseZetta10** [comment]
> I've merged few PR (Proxy PR included) to [another branch](https://github.com/c0re100/qBittorrent-Enhanced-Edition/tree/v4_5_merge). But I don't have proxy configure in qBittorrent, so I can't test it. Let's try~
> 
> Windows x64: https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.5.3.10/test_v4_5_merge_Win_x64.7z

I have tried the "proxy for RSS" and it worked fine
